validator/keymanager/types_test.go

```diff
@@ -20,3 +24,12 @@ var (
 	_ = keymanager.Deleter(&imported.Keymanager{})
 	_ = keymanager.Deleter(&derived.Keymanager{})
 )
+
+func TestKeystoreContainsPath(t *testing.T) {
+	keystore := keymanager.Keystore{}
+	encoded, err := json.Marshal(keystore)
+	want := "{\"crypto\":null,\"uuid\":\"\",\"pubkey\":\"\",\"version\":0,\"name\":\"\",\"path\":\"\"}"
+
+	require.NoError(t, err, "Unexpected error marshalling keystore")
+	require.Equal(t, want, string(encoded))
```

Comment:
I considered replacing this assertion with

```go
if !strings.Contains(string(encoded), "path") {
    t.Errorf("Encoded %s does not contain path", encoded)
}
```

which would remove line 31. Defer to reviewers on what they think is clearer

Reply:
`assert.Equal(t, true, strings.Contains(string(encoded), "path"))`